pyproject.toml

```diff
@@ -73,6 +74,10 @@ experimentalist-falsification =[
   "autora-experimentalist-falsification"
 ]
 
```

Comment:
```suggestion
```

Reply:
Yes, thanks -- just updated the repo! It's my first time publishing a package, so I didn't know I had to reflect releases in the repo too.